docs/reference/monitoring/troubleshooting.asciidoc

```diff
@@ -27,3 +30,14 @@ monitoring data by using {metricbeat} the indices have `-mb` in their names. If
 the indices do not exist, review your configuration. For example, see
 <<monitoring-production>>.
 
+[[monitoring-troubleshooting-uuid]]
+=== Some monitoring data is missing from {kib}
```

Comment:
I wonder if this is too vague, as it could apply to other situations as well, e.g. monitoring data is missing for some period of time. How about something along the lines of, "Monitoring data for some stack nodes or instances is missing from {kib}"

Reply:
Good point, thanks!